app/qml/ProjectWizardPage.qml

```diff
@@ -0,0 +1,203 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import QtGraphicalEffects 1.0
+import QtQuick.Dialogs 1.2
+import QgsQuick 0.1 as QgsQuick
+import lc 1.0
+import "."
+import "./components" // import InputStyle singleton
+Item {
+  id: projectWizardPanel
+
+  signal back
+
+  property real rowHeight: InputStyle.rowHeight
+  property var fontColor: InputStyle.fontColor
+  property var bgColor: InputStyle.clrPanelMain
+  property real panelMargin: 10 * QgsQuick.Utils.dp
+
+  property ListModel items: ListModel {}
+  property var supportedTypes: {
+    var types = __fieldsModel.supportedTypes()
+    for (var prop in types) {
+      projectWizardPanel.items.append({ "display": types[prop], "widget": prop })
+    }
+
+    projectWizardPanel.items
```

Comment:
Is `return` not needed?

Reply:
Implicit return works in QML, but only while defining property. Of course, `return` statement would work as well. I just prefer to use it without it similarly as I would define one-liner property 
[Small example if you would like](https://pastebin.com/KJ7RPezG)

Reply:
Cool, thanks :)